test/e2e/collection/fluentd/container_security_test.go

```diff
@@ -0,0 +1,70 @@
+package fluentd
+
+import (
+	"fmt"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/openshift/cluster-logging-operator/test/helpers"
+	"github.com/openshift/cluster-logging-operator/test/helpers/oc"
+)
+
+func checkMountReadOnly(mount string) {
+	touchFile := mount + "/1"
+	result, err := oc.Exec().WithNamespace(helpers.OpenshiftLoggingNS).Pod("service/fluentd").
+		WithCmd("bash", "-c", "touch "+touchFile).Run()
+	Expect(result).To(HavePrefix("touch: cannot touch '" + touchFile + "': Read-only file system"))
+	Expect(err).To(MatchError("exit status 1"))
+}
+
+var _ = Describe("Tests of fluentd container security stance", func() {
+	e2e := helpers.NewE2ETestFramework()
```

Comment:
Question more then anything... does this not have to be in a `var()` ?

Reply:
It doesn't _have_ to be